trappy/sched.py

```diff
@@ -66,9 +66,9 @@ def get_pids(self, key=""):
                                           pivot="cpu")
 """Load and Utilization Signals for CPUs"""
 
-SchedContribScaleFactor = register_dynamic_ftrace("SchedContribScaleFactor",
-                                                  "sched_contrib_scale_f:",
-                                                  "sched")
+SchedContribScaleF = register_dynamic_ftrace("SchedContribScaleF",
```

Comment:
Is this change done to make `SchedContribScaleF`'s `name` field the same as its `unique_word` field ? If so, why ? Is that actually required ? There are quite a few classes with different `name` and `unique_word` - at least most if not all of the thermal classes do (see [temperature](https://github.com/ARM-software/trappy/blob/master/trappy/thermal.py#L32) or [devfreq cooling stuff](https://github.com/ARM-software/trappy/blob/master/trappy/devfreq_power.py#L30).